test/js/web/workers/worker-fixture.ts

```diff
@@ -0,0 +1,8 @@
+declare var self: Worker;
```

Comment:
i guess `declare var` is enough to count as ts syntax. i imagined seeing some type annotation at all but this works.

Reply:
I can add some typing.